### ISSUE TITLE
The player markers (gang members, players, medics can not know where the players died) do not appear on the map

### ISSUE THREAD

**Issue by BraK3NN** [AsYetUntitled/Framework#441]

**Comment by imKLM** [comment]
what version of the files are you using? (if 5.0 give the commit hash for the most recent commit at the time of download/clone)

**Comment by Jawshy** [comment]
@BraK3NN, we need more information about your issue before we can help; fill in the [issue template](https://raw.githubusercontent.com/AsYetUntitled/Framework/master/.github/ISSUE_TEMPLATE.md) that was provided.

**Comment by Jason2605** [comment]
Not enough information to be able to reproduce the issue.

**Comment by Cpaztik** [comment]
Same issue for me, using 5.0, all markers do not appear.

**Comment by BoGuu** [comment]
Hi @Cpaztik,

Could you please create a new issue using the template provided, and any screenshots etc?
